docs/user/container.md

```diff
@@ -231,41 +246,129 @@ Access Control List   A::OWNER@:rwdtTaAo, A:G:GROUP@:rwtT
+| compression             | Yes             | Online compression off, or algorithm to use (off, lz4, deflate[1-4])|
+| dedup                   | Yes             | Inline deduplication off, or algorithm to use (hash or memcmp)|
 | dedup\_threshold        | Yes             | Minimum I/O size to consider for deduplication|
 | encryption              | Yes             | Inline encryption off, or algorithm to use (XTS[128,256], CBC[128,192,256] or GCM[128,256])|
-| status                  | No              | Current state of the container|
-| alloc\_oid              | No              | Maximum allocated object ID by container allocator|
 
-Refer to the [Data Integrity](#Data-Integrity) and [Access Control Lists](#Access_Control_Lists)
-sections for more details on the checksum and access-related properties.
+Please refer to the next sections for more details on each property.
+
+### Redundancy Factor
+
+Different objects in a DAOS container may belong to different object classes and
+have different levels of data protection. While this model gives a lot of control
+to the users, it also requires to carefully select a suitable class for each
+object. If objects with different data protection level are also stored in the
+same container, the user should also be prepared for the case where some objects
+might suffer from data loss after several cascading failures, while some others
+with higher level of data protection may not. This incurs extra complexity that
+not all I/O middleware necessarily wants to deal with.
+
+To lower the bar of adoption while still keeping the flexibility, two properties
+have been introduced:
+
+- the redundancy factor (rf) that describes the number of concurrent engine
+  exclusions that objects in the container are protected against. The rf value
+  is an integer between 0 (no data protection) and 5 (support up to 5
+  simultaneous failures).
+- a `health` property representing whether any object content might have been
+  lost due to cascading engine failures. The value of this property can be
+  either `HEALTHY` (no data loss) or `UNCLEAN` (data might have been lost).
+
+The redudancy factor can be set at container creation time and cannot be
+modified after creation.
+
+```bash
+$ daos cont create tank --label mycont1 --type POSIX --properties rf:1
+  Container UUID : b396e2ca-2077-4908-9ff2-1af4b4b2fd4a
+  Container Label: mycont1
+  Container Type : unknown
+Successfully created container b396e2ca-2077-4908-9ff2-1af4b4b2fd4a
+```
+
+It can be checked by listing the properties:
+
+```bash
+$ daos cont get-prop tank mycont1
+Properties for container mycont1
+Name                  Value
+----                  -----
+[...]
+Redundancy Factor     rf1
+Redundancy Level      rank (1)
+Health                HEALTHY
+[...]
+```
+
+Only objects with data protection enabled can be stored in such a container.
```

Comment:
Maybe:
Only objects with data protection enabled can be stored in such a container.
This includes replicated or erasure-coded objects. Attempts to open an object with an object class that does not support data redundancy (e.g., SX) will fail.

For rf2, only objects with at least 3-way replication or erasure coded with two parities or more can be stored in the container.